src/controllers/mainController.ts

```diff
@@ -1411,4 +1418,38 @@ export default class MainController implements vscode.Disposable {
 	public onClearAzureTokenCache(): void {
 		this.connectionManager.onClearTokenCache();
 	}
+
+	private ExecutionPlanCustomEditorProvider = class implements vscode.CustomTextEditorProvider {
```

Comment:
it may make sense to organize this differently, i.e., in it's own file in a ExecutionPlan directory or something like that.  But since it's a short code block and there's only one custom editor, this could be cleaned up later.